packages/opentelemetry-tracer-web/src/WebTracer.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BasicTracer, BasicTracerConfig } from '@opentelemetry/tracer-basic';
+import { ScopeManager } from '@opentelemetry/scope-base';
+import { StackScopeManager } from './StackScopeManager';
+/**
+ * WebTracerConfig provides an interface for configuring a Web Tracer.
+ */
+export interface WebTracerConfig extends Partial<BasicTracerConfig> {
+  scopeManager?: ScopeManager;
```

Comment:
`scopeManager`  is already defined in `BasicTracerConfig`. Why do you need this here, again?

Reply:
it is optional, in BasicTracerConfig it is not. So you can hook up later your own or for example `ZoneScopeManager` (TODO)

Reply:
Recently, we have made it as an optional param in `BasicTracerConfig`: https://github.com/open-telemetry/opentelemetry-js/blob/75a329776a72130ddda01a2d5f2541f40655ebef/packages/opentelemetry-tracer-basic/src/types.ts#L63